projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -732,6 +737,24 @@ describe('IgxDateRangePicker', () => {
                 fixture.detectChanges();
                 expect(fixture.componentInstance.dateRange.collapsed).toBeTruthy();
             }));
+
+            it('should properly set/update disabled when ChangeDetectionStrategy.OnPush is used', fakeAsync(() => {
```

Comment:
actually this testing if `interval` is working and correctly sets the component input. This does not test if picker's inputs are disabled. IMO you should test:
1. If when disabled is set to `true` inputs are disabled.
2. When change disabled run-time inputs change accordingly.

Both above in component with change detection strategy set to OnPush.